internal/controller/vrg_volsync.go

```diff
@@ -33,6 +33,9 @@ func (v *VRGInstance) restorePVsAndPVCsForVolSync() (int, error) {
 		failoverAction := v.instance.Spec.Action == ramendrv1alpha1.VRGActionFailover
 
```

Comment:
Did we lose the `protectedPVC` pointer updates in this function `restorePVsAndPVCsForVolSync`?

Reply:
Addressed as part of #1588